test/Microsoft.AspNetCore.Server.Kestrel.Performance/project.json

```diff
@@ -2,7 +2,8 @@
   "version": "1.0.0-*",
   "dependencies": {
     "BenchmarkDotNet": "0.10.0",
```

Comment:
Bump to 10.1 and netcoreapp1.1? as per https://github.com/aspnet/KestrelHttpServer/pull/1222/commits/8301721b80d57828fb106699685502c1d0ff5f58

Gives inbuilt accurate memory profiling